const index = [
  {
    category: 'Arabidopsis',
    title: 'Absence of DNA replication signal in DNA-Seq from sperm nuclei',
    accession: 'PRJNA914255',
    pmid: '',
    link: 'https://www.biorxiv.org/content/10.1101/2023.02.22.529524v1.full.pdf',
    author: 'Voichek Yoav et al.',
    time: '2023',
    study: 'https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA914255',
    'ref-genome': '',
    fastQC: '',
    abstract: 'Fertilization in Arabidopsis thaliana is a highly coordinated process that begins with a pollen tube delivering the two sperm cells into the embryo sac. Each sperm cell can then fertilize either the egg or the central cell to initiate embryo or endosperm development, respectively. The success of this double fertilization process requires a tight cell cycle synchrony between the male and female gametes to allow karyogamy (nuclei fusion). However, the cell cycle status of the male and female gametes during fertilization still remains elusive as DNA quantification and DNA replication assays have given conflicting results1–4. Here, to reconcile these results, we quantified the DNA replication state by DNA sequencing and performed microscopic analyses of fluorescent markers covering all the phases of the cell cycle. We show that male and female gametes in Arabidopsis are both arrested prior to DNA replication at maturity and initiate their DNA replication only during fertilization.'
  },
  {
    category: 'Arabidopsis',
    title: 'Hyponastic Leaves 1 protects pri-miRNAs from nuclear exosome attack',
    accession: 'PRJNA563051',
    pmid: '32636270',
    link: 'https://www.pnas.org/doi/full/10.1073/pnas.2007203117',
    author: 'Gao Shuai et al.',
    time: '2020',
    study: 'https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA563051',
    'ref-genome': '',
    fastQC: '',
    abstract: 'Biogenesis of plant microRNAs (miRNAs) takes place in nuclear dicing bodies (D-bodies), where the ribonulease III-type enzyme Dicer-like 1 (DCL1) processes primary transcripts of miRNAs (pri-miRNAs) into miRNA/miRNA* (*, passenger strand) duplexes from either base-to-loop or loop-to-base directions. Hyponastic Leaves 1 (HYL1), a double-stranded RNA-binding protein, is crucial for efficient and accurate processing. However, whether HYL1 has additional function remains unknown. Here, we report that HYL1 plays a noncanonical role in protecting pri-miRNAs from nuclear exosome attack in addition to ensuring processing. Loss of functions in SOP1 or HEN2, two cofactors of the nucleoplasmic exosome, significantly suppressed the morphological phenotypes of hyl1-2. Remarkably, mature miRNAs generated from loop-to-base processing were partially but preferentially restored in the hyl1 sop1 and hyl1 hen2 double mutants. Accordingly, loop-to-base-processed pri-miRNAs accumulated to higher levels in double mutants. In addition, dysfunction of HEN2, but not of SOP1, in hyl1-2 resulted in overaccumulation of many base-to-loop-processed pri-miRNAs, with most of their respective miRNAs unaffected. In summary, our findings reveal an antagonistic action of exosome in pri-miRNA biogenesis and uncover dual roles of HYL1 in stabilizing and processing of pri-miRNAs.Biogenesis of plant microRNAs (miRNAs) takes place in nuclear dicing bodies (D-bodies), where the ribonulease III-type enzyme Dicer-like 1 (DCL1) processes primary transcripts of miRNAs (pri-miRNAs) into miRNA/miRNA* (*, passenger strand) duplexes from either base-to-loop or loop-to-base directions. Hyponastic Leaves 1 (HYL1), a double-stranded RNA-binding protein, is crucial for efficient and accurate processing. However, whether HYL1 has additional function remains unknown. Here, we report that HYL1 plays a noncanonical role in protecting pri-miRNAs from nuclear exosome attack in addition to ensuring processing. Loss of functions in SOP1 or HEN2, two cofactors of the nucleoplasmic exosome, significantly suppressed the morphological phenotypes of hyl1-2. Remarkably, mature miRNAs generated from loop-to-base processing were partially but preferentially restored in the hyl1 sop1 and hyl1 hen2 double mutants. Accordingly, loop-to-base-processed pri-miRNAs accumulated to higher levels in double mutants. In addition, dysfunction of HEN2, but not of SOP1, in hyl1-2 resulted in overaccumulation of many base-to-loop-processed pri-miRNAs, with most of their respective miRNAs unaffected. In summary, our findings reveal an antagonistic action of exosome in pri-miRNA biogenesis and uncover dual roles of HYL1 in stabilizing and processing of pri-miRNAs.'
  },
  {
    category: 'Arabidopsis',
    title: 'High-quality Arabidopsis thaliana genome assembly with Nanopore and HiFi long reads',
    accession: 'PRJNA734474',
    pmid: '',
    link: 'https://www.biorxiv.org/content/10.1101/2021.06.08.447650v1.full',
    author: '',
    time: '',
    study: 'https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA734474',
    'ref-genome': '',
    fastQC: '',
    abstract: 'Here, we report a high-quality (HQ) and almost complete genome assembly with a single gap and quality value (QV) larger than 60 of the model plant Arabidopsis thaliana ecotype Columbia (Col-0), generated using combination of Oxford Nanopore Technology (ONT) ultra-long reads, high fidelity (HiFi) reads and Hi-C data. The total genome assembly size is 133,877,291 bp (chr1: 32,659,241 bp, chr2: 22,712,559 bp, chr3: 26,161,332 bp, chr4: 22,250,686 bp and chr5: 30,093,473 bp), and introduces 14.73 Mb (96% belong to centromere) novel sequences compared to TAIR10.1 reference genome. All five chromosomes of our HQ assembly are highly accurate with QV larger than 60, ranging from QV62 to QV68, which is significantly higher than TAIR10.1 referecne (44-51) and a recent published genome (41-43). We have completely resolved chr3 and chr5 from telomere-to-telomere. For chr2 and chr4, we have completely resolved apart from the nucleolar organizing regions, which are composed of highly long-repetitive DNA fragments. It has been reported that the length of centromere 1 is about 9 Mb and it is hard to assembly since tens of thousands of CEN180 satellite repeats. Based on the cutting-edge sequencing data, we assembled about 4Mb continuous sequence of centromere 1. We found different identity patterns across five centromeres, and all centromeres were significantly enriched with CENH3 ChIP-seq signals, confirming the accuracy of the assembly. We obtained four clusters of CEN180 repeats, and found CENH3 presented a strong preference for a cluster 3. Moreover, we observed hypomethylation patterns in CENH3 enriched regions. This high-quality assembly genome will be a valuable reference to assist us in the understanding of global pattern of centromeric polymorphism, genetic and epigenetic in naturally inbred lines of Arabidopsis thaliana.'
  },
  {
    category: 'Arabidopsis',
    title: 'Protein arginine methyltransferase 3 fine-tunes the assembly/disassembly of pre-ribosomes to repress nucleolar stress by interacting with RPS2B in arabidopsis',
    accession: 'PRJNA614590',
    pmid: '33069875',
    link: 'https://pubmed.ncbi.nlm.nih.gov/33069875/',
    author: 'Hang Runlai et al.',
    time: '2021',
    study: 'https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA614590',
    'ref-genome': '',
    fastQC: '',
    abstract: 'Ribosome biogenesis, which takes place mainly in the nucleolus, involves coordinated expression of pre-ribosomal RNAs (pre-rRNAs) and ribosomal proteins, pre-rRNA processing, and subunit assembly with the aid of numerous assembly factors. Our previous study showed that the Arabidopsis thaliana protein arginine methyltransferase AtPRMT3 regulates pre-rRNA processing; however, the underlying molecular mechanism remains unknown. Here, we report that AtPRMT3 interacts with Ribosomal Protein S2 (RPS2), facilitating processing of the 90S/Small Subunit (SSU) processome and repressing nucleolar stress. We isolated an intragenic suppressor of atprmt3-2, which rescues the developmental defects of atprmt3-2 while produces a putative truncated AtPRMT3 protein bearing the entire N-terminus but lacking an intact enzymatic activity domain We further identified RPS2 as an interacting partner of AtPRMT3, and found that loss-of-function rps2a2b mutants were phenotypically reminiscent of atprmt3, showing pleiotropic developmental defects and aberrant pre-rRNA processing. RPS2B binds directly to pre-rRNAs in the nucleus, and such binding is enhanced in atprmt3-2. Consistently, multiple components of the 90S/SSU processome were more enriched by RPS2B in atprmt3-2, which accounts for early pre-rRNA processing defects and results in nucleolar stress. Collectively, our study uncovered a novel mechanism by which AtPRMT3 cooperates with RPS2B to facilitate the dynamic assembly/disassembly of the 90S/SSU processome during ribosome biogenesis and repress nucleolar stress.'
  },
  {
    category: 'Arabidopsis',
    title: 'Novel regulators of sugar-mediated lateral root development in Arabidopsis thaliana',
    accession: 'PRJNA669402',
    pmid: '33113878',
    link: 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7716218/',
    author: 'Li Jinzhu et al.',
    time: '2020',
    study: 'https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA669402',
    'ref-genome': '',
    fastQC: '',
    abstract: 'In plants, cytosine DNA methylations (5mCs) can happen in three sequence contexts as CpG, CHG, and CHH (where H = A, C, or T), which play different roles in the regulation of biological processes. Although long Nanopore reads are advantageous in the detection of 5mCs comparing to short-read bisulfite sequencing, existing methods can only detect 5mCs in the CpG context, which limits their application in plants. Here, we develop DeepSignal-plant, a deep learning tool to detect genome-wide 5mCs of all three contexts in plants from Nanopore reads. We sequence Arabidopsis thaliana and Oryza sativa using both Nanopore and bisulfite sequencing. We develop a denoising process for training models, which enables DeepSignal-plant to achieve high correlations with bisulfite sequencing for 5mC detection in all three contexts. Furthermore, DeepSignal-plant can profile more 5mC sites, which will help to provide a more complete understanding of epigenetic mechanisms of different biological processes.'
  },
  {
    category: 'Arabidopsis',
    title: 'Genome-wide detection of cytosine methylations in plant from Nanopore data using deep learning',
    accession: 'PRJNA764549',
    pmid: '34645826',
    link: 'https://pubmed.ncbi.nlm.nih.gov/34645826/',
    author: 'Ni Peng et al.',
    time: '2021',
    study: 'https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA764549',
    'ref-genome': '',
    fastQC: '',
    abstract: 'In plants, cytosine DNA methylations (5mCs) can happen in three sequence contexts as CpG, CHG, and CHH (where H = A, C, or T), which play different roles in the regulation of biological processes. Although long Nanopore reads are advantageous in the detection of 5mCs comparing to short-read bisulfite sequencing, existing methods can only detect 5mCs in the CpG context, which limits their application in plants. Here, we develop DeepSignal-plant, a deep learning tool to detect genome-wide 5mCs of all three contexts in plants from Nanopore reads. We sequence Arabidopsis thaliana and Oryza sativa using both Nanopore and bisulfite sequencing. We develop a denoising process for training models, which enables DeepSignal-plant to achieve high correlations with bisulfite sequencing for 5mC detection in all three contexts. Furthermore, DeepSignal-plant can profile more 5mC sites, which will help to provide a more complete understanding of epigenetic mechanisms of different biological processes.'
  },
  {
    category: 'Arabidopsis',
    title: 'Arabidopsis thaliana cultivar:Columbia Genome sequencing and assembly',
    accession: 'PRJNA627807',
    pmid: '',
    link: '',
    author: '',
    time: '2020',
    study: 'https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA764549',
    'ref-genome': '',
    fastQC: '',
    abstract: 'We analyzed whole genome sequencing to detect genome rearrangement induced by room-temperature activated restriction enzyme.'
  },
  {
    category: 'Arabidopsis',
    title: 'RNase H1C collaborates with ssDNA binding proteins WHY1/3 and recombinase RecA1 to fulfill the DNA damage repair in Arabidopsis chloroplasts ',
    accession: 'PRJNA698745',
    pmid: '34133716',
    link: 'https://pubmed.ncbi.nlm.nih.gov/34133716/',
    author: 'Wang Wenjie et al.',
    time: '2021',
    study: 'https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA698745',
    'ref-genome': '',
    fastQC: '',
    abstract: 'Proper repair of damaged DNA is crucial for genetic integrity and organismal survival. As semi-autonomous organelles, plastids have their own genomes whose integrity must be preserved. Several factors have been shown to participate in plastid DNA damage repair; however, the underlying mechanism remains unclear. Here, we elucidate a mechanism of homologous recombination (HR) repair in chloroplasts that involves R-loops. We find that the recombinase RecA1 forms filaments in chloroplasts during HR repair, but aggregates as puncta when RNA:DNA hybrids accumulate. ssDNA-binding proteins WHY1/3 and chloroplast RNase H1 AtRNH1C are recruited to the same genomic sites to promote HR repair. Depletion of AtRNH1C or WHY1/3 significantly suppresses the binding of RNA polymerase to the damaged DNA, thus reducing HR repair and modulating microhomology-mediated double-strand break repair. Furthermore, we show that DNA polymerase IB works with AtRNH1C genetically to complete the DNA damage repair process. This study reveals the positive role of R-loops in facilitating the activities of WHY1/3 and RecA1, which in turn secures HR repair and organellar development.'
  },
  {
    category: 'Arabidopsis',
    title: 'An ABA-GA bistable switch can account for natural variation in the variability of Arabidopsis seed germination Time',
    accession: 'PRJNA486286',
    pmid: '34059197',
    link: 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8169117/',
    author: 'Abley Katie et al.',
    time: '2021',
    study: 'https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA486286',
    'ref-genome': '',
    fastQC: '',
    abstract: 'Genetically identical plants growing in the same conditions can display heterogeneous phenotypes. Here we use Arabidopsis seed germination Time： as a model system to examine phenotypic variability and its underlying mechanisms. We show extensive variation in seed germination Time： variability between Arabidopsis accessions and use a multiparent recombinant inbred population to identify two genetic loci involved in this trait. Both loci include genes implicated in modulating abscisic acid (ABA) sensitivity. Mutually antagonistic regulation between ABA, which represses germination, and gibberellic acid (GA), which promotes germination, underlies the decision to germinate and can act as a bistable switch. A simple stochastic model of the ABA-GA network shows that modulating ABA sensitivity can generate the range of germination Time： distributions we observe experimentally. We validate the model by testing its predictions on the effects of exogenous hormone addition. Our work provides a foundation for understanding the mechanism and functional role of phenotypic variability in germination Time.'
  },
  {
    category: 'Arabidopsis',
    title: 'Genome sequencing of the Arabidopsis thaliana PSB-D cell line',
    accession: 'PRJNA638606',
    pmid: '',
    link: '',
    author: 'Abley Katie et al.',
    time: '2020',
    study: 'https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA638606',
    'ref-genome': '',
    fastQC: '',
    abstract: 'PSB-D is a dark-grown cell line originally developed as a synchronized system for the study of the cell cycle in plants. It has also been used as an experimental system for proteomics and other work. It is important to understand the system-wide molecular characteristics that distinguish PSB-D from other Arabidopsis tissue types in order to better interpret results and understand the limitations of the system, as well as to provide accurate protein sequence databases for LC-MS/MS search. Genome re-sequencing of PSB-D is part of an extensive genomic, transcriptomic and proteomic effort to characterize the cell line for these purposes.'
  },
  {
    category: 'Arabidopsis thaliana',
    title: 'DNA Methylation shapes histone modification landscape for transcription regulation in Arabidopsis',
    accession: 'GSE183957',
    pmid: '36127735',
    link: 'https://pubmed.ncbi.nlm.nih.gov/36127735/',
    author: '',
    time: '',
    study: 'https://www.ncbi.nlm.nih.gov/bioproject/?term=GSE183957',
    'ref-genome': '',
    fastQC: '',
    abstract: 'DNA methylation and histone modifications cooperate to shape the epigenetic landscape for transcription regulation, but their orchestration is not fully understood due to the lack of met1 drm1 drm2 cmt2 cmt3 mutants (mddcc), in which total DNA methylation were eliminated. This is the first time to generate mddcc that allow us to elucidate the roles of DNA methylation in the upheaval of histone modification landscape, transposon activity and gene expression. We found that DNA methylation elimination-triggered genome-wide transcriptional reprogramming is largely orchestrated by the redistribution and conversion of histone modifications. We also distinguished the contribution of CG and non-CG methylation to the histone modification landscape. Furthermore, we reveal histone modifications can serve as an independent regulatory system despite the influence of DNA methylation. Taken together, we provide comprehensive insights into the roles of DNA methylation in shaping epigenomic landscape for genome transcription, which deep understanding of epigenomic regulation.'
  },
  {
    category: 'Arabidopsis thaliana',
    title: 'The DREAM complex antagonizes WDR5a and represses the productive elongation of transcription in Arabidopsis',
    accession: 'GSE181486',
    pmid: '35759663',
    link: 'https://pubmed.ncbi.nlm.nih.gov/35759663/',
    author: '',
    time: '',
    study: 'https://www.ncbi.nlm.nih.gov/bioproject/?term=GSE181486',
    'ref-genome': '',
    fastQC: '',
    abstract: 'The master transcriptional repressor DREAM (dimerization partner, RB-like, E2F and multi-vulval class B) complex regulates cell cycle in eukaryotes, but much remains unknown about how it transmits repressive signals on chromatin to the primary transcriptional machinery (e.g., Pol II). Through a genetic screen, we identified BTE1 (barrier of transcription elongation 1), a key component in the plant DREAM complex. The subsequent characterization demonstrated that the DREAM complex represses Pol II elongation. The core DREAM subunits E2F transcription factors recruit BTE1 to accumulate at the promoter-proximal regions of target genes. DREAM target genes exhibit characteristic enrichment of H2A.Z and H3K4me2 modification on chromatin. The DREAM complex BTE1 subunit interacts with WDR5a, repressing WDR5a chromatin binding and the productive elongation of transcription on BTE1-targeted genomic loci. Thus, we demonstrate a major role of DREAM complex in the control of transition from transcription initiation to productive elongation around TSSs on the open chromatin via inhibiting WDR5a function.'
  },
  {
    category: 'Arabidopsis thaliana',
    title: 'REVEILLE5 buffers the activity of CIRCADIAN CLOCK ASSOCIATED1 to modulate thermoresponsive hypocotyl growth in Arabidopsis',
    accession: 'GSE180381',
    pmid: '35603836',
    link: 'https://pubmed.ncbi.nlm.nih.gov/35603836/',
    author: '',
    time: '',
    study: 'https://www.ncbi.nlm.nih.gov/bioproject/?term=GSE180381',
    'ref-genome': '',
    fastQC: '',
    abstract: 'To further understand how RVE5 controls thermo-responses, we generated RVE5-MYC overexpression plants with the CaMV 35S promoter. Using these RVE5-MYC overexpression plants grown at 29°C, we performed Chromatin Immunoprecipitation-Sequencing'
  },
  {
    category: 'Arabidopsis thaliana',
    title: 'A REF6-dependent H3K27me3-depleted state facilitates gene activation during germination in Arabidopsis [H3K27me3]',
    accession: 'GSE167503',
    pmid: '36113770',
    link: 'https://pubmed.ncbi.nlm.nih.gov/36113770/',
    author: '',
    time: '',
    study: 'https://www.ncbi.nlm.nih.gov/bioproject/?term=GSE167503',
    'ref-genome': '',
    fastQC: '',
    abstract: 'We report transcriptome changes and genome-wide dynamics of H3K27me3 during seed germination in Arabidopdsis, and investigate the impact of REF6-mediated H3K27 demethylation on germination. Compared with transcriptome changes, we discover delayed H3K27me3 reprogramming closely associated with the embryo to vegetative cell fate switch. REF6-mediated H3K27 demethylation promotes germination but does not significantly contribute to H3K27me3 dynamics during germination, but rather stably establishes an H3K27me3-depleted state permissive to transcription. By analyzing REF6 genomic binding, we show that it is absent from mature embryo chromatin and gradually establishes occupancy during the course of germination to counteract increased PRC2 activity.'
  },
  {
    category: 'Arabidopsis thaliana',
    title: 'Genome-widebinding maps of arabidopsis WRKY63',
    accession: 'GSE177884',
    pmid: '35708655',
    link: 'https://pubmed.ncbi.nlm.nih.gov/35708655/',
    author: '',
    time: '',
    study: 'https://www.ncbi.nlm.nih.gov/bioproject/?term=GSE177884',
    'ref-genome': '',
    fastQC: '',
    abstract: 'We report the application of single-molecule-based sequencing technology for high-throughput profiling of WRKY63 in 10 days old pro::WRKY63:GFP arabidopsis. By obtaining sequence from chromatin immunoprecipitated DNA, we mapped genome-wide binding levels of WRKY63:GFP. ChIP was performed using anti-GFP antibody (ab290), and ChIP DNA were analyzed by Illumina NovoSeq.'
  },
  {
    category: 'Arabidopsis thaliana',
    title: 'Deceleration of the cell cycle underpins a switch from proliferative to terminal division in plant stomatal lineage',
    accession: 'GSE173338',
    pmid: '35148836',
    link: 'https://pubmed.ncbi.nlm.nih.gov/35148836/',
    author: '',
    time: '',
    study: 'https://www.ncbi.nlm.nih.gov/bioproject/?term=GSE173338',
    'ref-genome': '',
    fastQC: '',
    abstract: 'We have previously identified genes that are induced by MUTE, key transcription factor for stomatal lineage fate commitment and symmetric cell division that create stomata surrounded by a pair of guard cells in Arabidopsis. To test whether these genes are indeed direct MUTE targets, we performed genome-wide MUTE ChIP-sequencing.'
  },
  {
    category: 'Arabidopsis thaliana',
    title: 'Interplay between gene expression, histone modifications, and DNA methylation mediated by miR778 during beet cyst nematode parasitism of Arabidopsis',
    accession: 'GSE173812',
    pmid: '35579365',
    link: 'https://pubmed.ncbi.nlm.nih.gov/35579365/',
    author: '',
    time: '',
    study: 'https://www.ncbi.nlm.nih.gov/bioproject/?term=GSE173812',
    'ref-genome': '',
    fastQC: '',
    abstract: 'Purpose: The aim of this study is to investigate the role of H3K9 methylation in the development of syncytia induced by the beet cyst nematode (BCN, Heterodera schachtii). For this, ChIP-seq analysis was conducted in root tissues of Col-0 wild-type versus single suvh5 and suvh6 mutant plants and under BCN-infected and noninfected conditions. Methods: Col-0 wild-type and suvh5 and suvh6 mutant plants were grown and 10-day-old plants were inoculated with 100 second-stage juvenile (J2) H. schachtii at 10 days old. Five days post inoculation, root tissues were collected from infected and noninfected plants in triplicate. ChIP DNA was precipitated using the H3k9me2 antibody (Abcam) and prepared into libraries. Then, high throughput ChIP DNA sequencing was performed using the Illumina NovoSeq 6000. Raw reads were trimmed of adapter sequences and overrepresented reads. Then, high quality, paired-end reads were aligned to the Arabidopsis thaliana reference genome (TAIR10.28) using Bowtie2. Finally, histone peaks were called using MACS2 for the identification of genes associated with differentially dimethylated H3K9 in wild-type versus mutant roots under BCN-infected and noninfected conditions. Results and Conclusions: Under noninfected conditions, 511 and 143 H3K9me2 peaks were enriched in Col-0 compared to suvh5 and suvh6, respectively. The majority of these H3K9me2 peaks are located within TEs, which is consistent with previous findings that H3K9me2 corresponds to the repression of detrimental TEs. Our results suggest, however, that H3K9me2 adopts a new role in response to BCN. BCN-infected Col-0 had 791 H3K9me2 peaks compared to noninfected Col-0, 743 of which are located in the promoter or body of protein-coding genes. These peaks are associated with 1180 protein-coding genes. Similarly, suvh5 and suvh6 single mutants had 353 and 3106 H3K9me2 peaks compared to BCN-infected Col-0, respectively. The majority of these peaks are located in protein-coding genes and are associated with 558 and 4594 protein-coding genes, respectively. Interestingly, about 65% of peaks enriched in the BCN-infected wild type overlapped with peaks depleted in the suvh6 mutant, indicating that SUVH6 contributes to the majority of H3K9me2 during BCN parasitism.'
  },
  {
    category: 'Arabidopsis thaliana',
    title: 'The SNF2 ATPase-independent function of the INO80 chromatin remodeler',
    accession: 'GSE162374',
    pmid: '34270751',
    link: 'https://pubmed.ncbi.nlm.nih.gov/34270751/',
    author: '',
    time: '',
    study: 'https://www.ncbi.nlm.nih.gov/bioproject/?term=GSE162374',
    'ref-genome': '',
    fastQC: '',
    abstract: 'In the multi-subunit INO80 chromatin-remodeling complex, all the auxiliary subunits assemble on three distinct domains of the catalytic chromatin remodeler INO80, which are N-terminal domain, HSA domain, and ATPase domain. While the ATPase and HSA domains and the auxiliary subunits assembling on the domains are known to be responsible for ATP hydrolysis and chromatin remodeling, it is largely unknown how the auxiliary subunits assembling on the INO80 N-terminal domain regulate the chromatin status. We identify both conserved and non-conserved auxiliary subunits of the INO80 complex in Arabidopsis thaliana. All the auxiliary subunits assemble on the conserved N-terminal domain, HSA domain, and ATPase domain of INO80 in Arabidopsis. While the auxiliary subunits assembling on the INO80 ATPase domain are required for the ATPase-dependent function, the INO80 N-terminal domain and the auxiliary subunits assembling on the domain can regulate gene expression and development even when the ATPase domain is absent, suggesting that INO80 has an ATPase-independent role. Furthermore, we find that a subclass of the COMPASS histone H3K4 methyltransferase complexes assemble on the INO80 N-terminal domain in the INO80 complex and function together with the other auxiliary subunits assembling on the INO80 N-terminal domain, thereby facilitating the ATPase-independent function. This study suggests that the conserved chromatin remodeler INO80 has an ATPase-independent role and demonstrates that the auxiliary subunits assembling on the INO80 N-terminal domain are required for the ATPase-independent role.'
  },
  {
    category: 'Arabidopsis thaliana',
    title: 'Genome-wide H3K27me3 analysis in Arabidopsis',
    accession: 'GSE185120',
    pmid: '',
    link: '',
    author: '',
    time: '',
    study: 'https://www.ncbi.nlm.nih.gov/bioproject/?term=GSE185120',
    'ref-genome': '',
    fastQC: '',
    abstract: 'To explore the bivalent histone modifications in the Arabidopsis genome, we examined genome-wide histone 3 lysine-27 trimethylation (H3K27me3) and histone 3 lysine-4 trimethylation (H3K4me3) in 5-day-old seedlings (Col-0) by ChIP-seq. We found that more than 1300 genes loci contain both H3K27me3 and H3K4me3.'
  },
  {
    category: 'Arabidopsis thaliana',
    title: 'Genome-wide H3K27me3 analysis in Arabidopsis',
    accession: 'GSE185121',
    pmid: '',
    link: '',
    author: '',
    time: '',
    study: 'https://www.ncbi.nlm.nih.gov/bioproject/?term=GSE185121',
    'ref-genome': '',
    fastQC: '',
    abstract: 'To explore the bivalent histone modifications in the Arabidopsis genome, we examined genome-wide histone 3 lysine-27 trimethylation (H3K27me3) and histone 3 lysine-4 trimethylation (H3K4me3) in 5-day-old seedlings (Col-0) by ChIP-seq. We found that more than 1300 genes loci contain both H3K27me3 and H3K4me3.'
  },
  {
    category: 'Arabidopsis thaliana',
    title: 'H3K9 demethylation-mediated transcriptional activation is required for male meiosis in Arabidopsis thaliana',
    accession: 'GSE186532',
    pmid: '35638341',
    link: 'https://pubmed.ncbi.nlm.nih.gov/35638341/',
    author: '',
    time: '',
    study: 'https://www.ncbi.nlm.nih.gov/bioproject/?term=GSE186532',
    'ref-genome': '',
    fastQC: '',
    abstract: 'Dimethylation at histone H3 lysine 9 (H3K9me2), a critical mark for heterochromatin formation and transcriptional silencing, is usually methylated at transposable elements and repetitive sequences but unmethylated at bodies of protein-coding genes. It is known that deposition of H3K9me2 is essential for proper prophase progression in both mammals and plants. Nevertheless, function of removing H3K9me2 for meiosis is unknown. In this study, we show that H3K9 demethylases, IBM1 and JMJ27, together regulate chromosome pairing and segregation in early meiosis, through protecting thousands of protein-coding genes from ectopic H3K9me2, including meiotic-essential genes. In addition to IBM1 and JMJ27, we also found that the cohesin complex cofactors Precocious Dissociation of Sisters 5 (PDS5s) regulate the expression of essential meiotic genes by interacting with H3K9 demethylases, acting downstream of H3K9 demethylation. Therefore, H3K9 demethylases and PDS5s coordinately regulate male meiosis through regulating the expression of meiotic-essential genes. These findings provide insights into a novel function of removing H3K9me2 for meiosis and deeper understanding of the H3K9 demethylation-mediated transcriptional activation pathway.'
  },
  {
    category: 'Arabidopsis thaliana',
    title: 'Genome-wide maps of chromatin state in Col-0, jmj28 mutant, and atx1/2 mutant',
    accession: 'GSE188134',
    pmid: '36827182',
    link: 'https://pubmed.ncbi.nlm.nih.gov/36827182/',
    author: '',
    time: '',
    study: 'https://www.ncbi.nlm.nih.gov/bioproject/?term=GSE188134',
    'ref-genome': '',
    fastQC: '',
    abstract: 'Arabidopsis thaliana CONSTANS (CO) is an essential transcription factor that promotes flowering by activating the expression of the floral integrator FLOWERING LOCUS T (FT). A number of histone modification enzymes involved in the regulation of flowering have been identified, but the involvement of epigenetic mechanisms in the regulation of the core flowering regulator CO remains unclear. Previous studies have indicated that the transcription factors, FLOWERING BHLH1 (FBH1), FBH2, FBH3, and FBH4, function redundantly to activate the expression of CO. In this study, we found that the KDM3 group H3K9 demethylase JMJ28 interacts with the FBH transcription factors to activate CO by removing the repressive mark H3K9me2. The occupancy of JMJ28 on the CO locus is decreased in the fbh quadruple mutant, suggesting that the binding of JMJ28 is dependent on FBHs. Furthermore, genome-wide occupancy profile analyses indicate that the binding of JMJ28 to the genome overlaps with that of FBH3, indicating a functional association of JMJ28 and FBH3. Together, these results indicate that Arabidopsis JMJ28 functions as a CO activator by interacting with the FBH transcription factors to remove H3K9me2 from the CO locus.'
  },
  {
    category: 'Arabidopsis thaliana',
    title: 'Genome-wide occupancy of JMJ28 in Arabidopsis',
    accession: 'GSE188139',
    pmid: '36827182',
    link: 'https://pubmed.ncbi.nlm.nih.gov/36827182/',
    author: '',
    time: '',
    study: 'https://www.ncbi.nlm.nih.gov/bioproject/?term=GSE188139',
    'ref-genome': '',
    fastQC: '',
    abstract: 'Arabidopsis thaliana CONSTANS (CO) is an essential transcription factor that promotes flowering by activating the expression of the floral integrator FLOWERING LOCUS T (FT). A number of histone modification enzymes involved in the regulation of flowering have been identified, but the involvement of epigenetic mechanisms in the regulation of the core flowering regulator CO remains unclear. Previous studies have indicated that the transcription factors, FLOWERING BHLH1 (FBH1), FBH2, FBH3, and FBH4, function redundantly to activate the expression of CO. In this study, we found that the KDM3 group H3K9 demethylase JMJ28 interacts with the FBH transcription factors to activate CO by removing the repressive mark H3K9me2. The occupancy of JMJ28 on the CO locus is decreased in the fbh quadruple mutant, suggesting that the binding of JMJ28 is dependent on FBHs. Furthermore, genome-wide occupancy profile analyses indicate that the binding of JMJ28 to the genome overlaps with that of FBH3, indicating a functional association of JMJ28 and FBH3. Together, these results indicate that Arabidopsis JMJ28 functions as a CO activator by interacting with the FBH transcription factors to remove H3K9me2 from the CO locus.'
  },
]

/**
 * @name getSpecies
 * @description extract the public species in study array above
 * @returns { Array<string> } species list
 */
export const getSpecies = () => {
  const speciesSet = new Set()
  index.forEach(i => {
    speciesSet.add(i.category)
  })
  const species = Array.from(speciesSet)
  
  return species
}

export default index